### PR TITLE
added assignment of phase 1 crypto

### PR DESCRIPTION
**Pull request by ceekay8u**

### CODE REVIEW

File: RE-Cryptography/Sessions-5-&-6/krishna/krishna-ceaser_encryption.c [hunk 1]
```diff
@@ -0,0 +1,23 @@
+#include <stdio.h>
+
+void main()
+{
+    char str[500];
+    FILE* f1 = fopen("/home/akash/akash-message-caesar.txt","r");
```
Comment (mahirjain25) [inline RE-Cryptography/Sessions-5-&-6/krishna/krishna-ceaser_encryption.c]: Indicates that you did not do this assignment in true honesty. Sad to see. Write original code.

Comment (mahirjain25) [reply RE-Cryptography/Sessions-5-&-6/krishna/krishna-ceaser_encryption.c]: You are supposed to do Phase 1 individually

**Comment by nimbus98** [comment]
Bruh..

File: RE-Cryptography/Sessions-5-&-6/krishna/krishna_ceaser_cipher.txt [hunk 2]
```diff
@@ -1 +1 @@
-���'Ϸ�ý'��ѫ�˱'Ϸ�'�ÿ�'�ñ�'շ�'��ϱ�C'���'Ϸ�ý'���'���'���'ų�'���ϷC'F��'Ϸ��'Ǳ���'���'���'��ſ�ͷ'�ï'Ϸ�ѵ�Ͽ���C'M�'���'����'����ñ?'����ñ'��ͱ'չ��'����'���CCX���'���˱�'��ï�'��'�ŵ�Ϸ��'CCǩ��
\ No newline at end of file
+���'Ϸ�ý'��ѫ�˱'Ϸ�'�ÿ�'�ñ�'շ�'��ϱ�C'���'Ϸ�ý'���'���'���'ų�'���ϷC'I��'Ϸ��'Ǳ���'���'���'��ſ�ͷ'�ï'Ϸ�ѵ�Ͽ���C'P�'���'����'����ñ?'����ñ'��ͱ'չ��'����'���CC����'���˱�'��ï�'��'�ŵ�Ϸ��'CCǩ��
```
Comment (mahirjain25) [inline RE-Cryptography/Sessions-5-&-6/krishna/krishna_ceaser_cipher.txt]: Why does this CT have so many unknown characters?
Regenerate. Start with Phase 2 also, your PR will be merged once you finish both only now.

**Comment by ShashankP19** [comment]
@ceekay8u Your substitution cipher text should contain on lowercase alphabets ('?' shouldn't be there). Rest seems fine.

**Pull request merged**